### DISCUSSION TITLE
why my account mark spam?

### DISCUSSION THREAD

**Discussion by camerahadong** [community/community#86111] [Other Feature Feedback, Questions, & Ideas]
I have fixed it, thank you

**Comment by Shahnawazk7944** [comment]
There could be several reasons why your GitHub account is marked as spam. Some possible reasons include:

1. **Excessive or automated activity:** If your account is engaging in a large volume of automated or suspicious activity, it may trigger spam detection algorithms.

3. **Violation of GitHub's terms of service:** If your account is found to be violating GitHub's terms of service, such as by posting malicious content or engaging in abusive behavior, it may be marked as spam.

5. **Reports from other users:** If other users have reported your account as spam, it may trigger a review by GitHub's moderation team.

If your account has been marked as spam, you should reach out to GitHub's support team for assistance in resolving the issue. They can provide more specific information about why your account was flagged and help you address any underlying issues.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @camerahadong ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you.

**Comment by tsvtln** [comment]
how did you fix it?

**Comment by Snickers-khalil** [comment]
haw to fix it

**Comment by chrisstef** [comment]
@queenofcorgis It's been 1 month after my account was flagged as spam, for no obvious reason, and this has already costed me 2 job applications where I couldn't showcase my codebase and give access to a 3rd party app required for the application process. Github support hasn't reached out to me yet. Could I also ask for your help?

**Comment by edwardfernandes** [comment]
I opened a ticket on February 3rd and so far I haven't had any response and being marked as "spammy" is already affecting my workflow too much. I need a solution, urgently.

https://support.github.com/ticket/personal/0/3213804

**Comment by LK-langke** [comment]
Mee too
